contrib/babelfishpg_common/src/encoding/mb/conv.c

```diff
@@ -402,8 +402,8 @@ TsqlLocalToUtf(const unsigned char *iso, int len,
 		unsigned char b4 = 0;
 
 		/* "break" cases all represent errors */
-		if (*iso == '\0')
-			break;
+		// if (*iso == '\0')
```

Comment:
Have a test to check if it resolves the failure for NULL byte sequences which we were facing earlier during convert.